iina/Extensions.swift

```diff
@@ -482,3 +482,9 @@ extension URL {
     return (try? self.resourceValues(forKeys: [.isDirectoryKey]))?.isDirectory ?? false
   }
 }
+
+extension NSPasteboard.PasteboardType {
+  static let nsURL = NSPasteboard.PasteboardType("NSURL")
```

Comment:
I'd argue that this should be `static let URL = NSPasteboard.PasteboardType("NSURL")`. `NSString` has already been migrated by Apple as `string` so I think dropping the prefix would be more consistent.

Reply:
`NSPasteboard.PasteboardType` already has a member `url` for "public.url" (and `filenames` for "public.filenames"), so `nsURL` would be appropriate if we want to refer to "NSURL" clearly, as well as to avoid name collision.